src/account_multisig.ts

```diff
@@ -36,10 +58,24 @@ let storageFallback = {
 export class AccountMultisig extends Account {
     public contract: MultisigContract;
     public storage: any;
+    public sendCode: sendCodeFunction;
+    public getCode: getCodeFunction;
+    public verifyCode: verifyCodeFunction;
+    public onResult: Function;
 
-    constructor(connection: Connection, accountId: string, storage: any) {
+    /********************************
+    AccountMultisig has options object where you can provide callbacks for:
+    - sendCode: how to send the 2FA code in case you don't use NEAR Contract Helper
```

Comment:
@mattlockyer this seems to be very specific to our 2FA implementation vs multisig in general. Should we split into 2 classes with `Account2FA` extending `AccountMultsig`?

Reply:
(at the very least we should rename, so that it's possible to introduce `AccountMultsig` in follow-up without breaking API